source/NVDAObjects/window/winword.py

```diff
@@ -1468,42 +1242,39 @@ def script_moveParagraphUp(self,gesture):
 				ui.message(_("Moved above blank paragraph"))
 
 	def script_increaseDecreaseOutlineLevel(self,gesture):
+		if not self.WinwordSelectionObject:
+			# No Object model, so just let the gesture through
+			return gesture.send()
 		val=self._WaitForValueChangeForAction(lambda: gesture.send(),lambda: self.WinwordSelectionObject.paragraphFormat.outlineLevel)
 		style=self.WinwordSelectionObject.style.nameLocal
 		# Translators: the message when the outline level / style is changed in Microsoft word
 		ui.message(_("{styleName} style, outline level {outlineLevel}").format(styleName=style,outlineLevel=val))
 
 	def script_increaseDecreaseFontSize(self,gesture):
+		if not self.WinwordSelectionObject:
+			# No Object model, so just let the gesture through
+			return gesture.send()
 		val=self._WaitForValueChangeForAction(lambda: gesture.send(),lambda: self.WinwordSelectionObject.font.size)
 		# Translators: a message when increasing or decreasing font size in Microsoft Word
 		ui.message(_("{size:g} point font").format(size=val))
 
-	def script_caret_moveByCell(self,gesture):
-		gesture.send()
-		info=self.makeTextInfo(textInfos.POSITION_SELECTION)
-		inTable=info._rangeObj.tables.count>0
-		isCollapsed=info.isCollapsed
-		if inTable:
-			info.expand(textInfos.UNIT_CELL)
-			speech.speakTextInfo(info,reason=controlTypes.REASON_FOCUS)
-			braille.handler.handleCaretMove(self)
-
 	def script_tab(self,gesture):
 		gesture.send()
+		selectionObj=self.WinwordSelectionObject
 		info=self.makeTextInfo(textInfos.POSITION_SELECTION)
-		inTable=info._rangeObj.tables.count>0
+		inTable=selectionObj.tables.count>0 if selectionObj else False
 		isCollapsed=info.isCollapsed
 		if inTable and isCollapsed:
-			info.expand(textInfos.UNIT_CELL)
+			info.expand(textInfos.UNIT_PARAGRAPH)
 			isCollapsed=False

```

Comment:
This seems like an optimisation, but could this cause a bug if the call to `expand` failed for some reason? Should we not check `info.isCollapsed`? to be sure?